linode/placementgroup/framework_models.go

```diff
@@ -0,0 +1,73 @@
+package placementgroup
+
+import (
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/linode/linodego"
+	"github.com/linode/terraform-provider-linode/v2/linode/helper"
+)
+
+type PlacementGroupMemberModel struct {
+	LinodeID    types.Int64 `tfsdk:"linode_id"`
+	IsCompliant types.Bool  `tfsdk:"is_compliant"`
+}
+
+func (m *PlacementGroupMemberModel) FlattenPlacementGroupMember(member *linodego.PlacementGroupMember) {
+	// This object is always computed so we don't need to respect preserveKnown here.
+	// It may still be good practice to respect preserveKnown here though, so definitely
```

Comment:
While I think it's completely fine to not respect `preserveKnown` for a fully computed attribute, I recently realize that when we have to determine whether it's an unknown, we can do conversions between a slice of model structs and a Framework's `types.List`:
```go
list := types.ListValueFrom(ctx, someObjectType, sliceOfModels)

list.ElementsAs(ctx, &sliceOfModels, false)
```

https://github.com/linode/terraform-provider-linode/pull/1405/files#diff-b51120d512a9186537c26eccf08af488e064b5ddb2ee15e98f01627c6317739d

Reply:
Good to know!